clients/cli/src/find_pdas.rs

```diff
@@ -22,6 +23,10 @@ pub struct FindPdasArgs {
     #[clap(value_parser = parse_pubkey)]
     pub unwrapped_mint: Pubkey,
 
+    /// The address of the token program that the unwrapped tokens belong to
+    #[clap(value_parser = parse_token_program)]
+    pub unwrapped_token_program: Pubkey,
+
```

Comment:
This can be done in follow-up work, but it could be nice to *not* require this argument, and use RPC to figure out the unwrapped token program id.

Reply:
Indeed, that's a more helpful API. Simple enough to fix here 👍